certimporter/CertImporter/CertImporter.cpp

```diff
@@ -145,8 +145,10 @@ int wmain(int argc, wchar_t *argv[], wchar_t *envp[])
 
 	// Figure out which action to take
 	int(*actionFn)(HCERTSTORE, LPCWSTR);
+	DWORD additionalOpenFlags = 0;
 	if (wcsncmp(action, L"find", 4) == 0) {
 		actionFn = checkExists;
+		additionalOpenFlags = CERT_STORE_READONLY_FLAG;

```

Comment:
This allows us to inspect existing certs without prompting for privilege escalation.